assets/aml-benchmark/validation_rules.json

```diff
@@ -0,0 +1,13 @@
+{
+  "pep8": {
+    "ignore": [
+      "W503",
+      "E402"
+    ],
+    "justification": [
+      "W503: W503 and W504 are mutually exclusive. W503 is more widely used, so ignoring it.",
+      "E402: All module level imports can't be before `sys.path.append(os.getcwd())`."
```

Comment:
Hi @vizhur, I will get rid of W503 as this was for older version and I don't see this being flagged now. E402 will stay because this is required to support the code structure that we have. Let me know if it's ok to live with this, will update to only ignore E402 in specific files as done in this [vallidation_rules.json](https://github.com/Azure/azureml-assets/blob/main/assets/benchmark/validation_rules.json)